boxoffice/extapi/slack.py

```diff
@@ -0,0 +1,28 @@
+from boxoffice import db, app
+from boxoffice.models import ItemCollection, LineItem, Order, ORDER_STATUS
+from datetime import datetime as dt
+import requests
+from urllib import urlencode
+import json
+from tabulate import tabulate
+from rq.decorators import job
+
+@job('boxoffice')
+def post_stats(order_type, id, webhook_url):
+    if id is None:
+        return None
```

Comment:
Don't think you need these two lines. Let it fail if `id` isn't passed. It's a mandatory argument anyway.